doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -166,10 +175,29 @@ Modem library
 * The AT socket API is now deprecated.
 * The DFU socket API is now deprecated.
 
+Common Application Framework (CAF)
+----------------------------------
+
+* Added :ref:`caf_net_state`.
+* Added :ref:`caf_power_manager`.
+
+Profiler
+--------
+
+* Added profiling string data.
+* Optimized numeric data encoding.
+
+Edge Impulse
+------------
+
+* Added Thingy:53 support to :ref:`nrf_machine_learning_app`.
+* Added nRF52840DK configuration with data forwarder over NUS to :ref:`nrf_machine_learning_app`.
+
 Pelion
 ------
 
 * Updated Pelion Device Management Client library version to 4.10.0.
+* Switched to using :ref:`caf_power_manager` and :ref:`caf_net_state`.
```

Comment:
Isn't this valid only for the Pelion Client app?

Reply:
Yes, updated.